meshroom/core/attribute.py

```diff
@@ -446,6 +449,28 @@ def updateInternals(self):
         # Emit if the enable status has changed
         self.setEnabled(self.getEnabled())
 
+    def _is3D(self) -> bool:
+        """ Return True if the current attribute is considered as a 3d file """
+
+        # List of supported extensions, taken from Viewer3DSettings
```

Comment:
Leftover comment, to be removed since the list of supported extensions has been moved upwards.
```suggestion

```